df_py/challenge/csvs.py

```diff
@@ -83,7 +83,7 @@ def load_challenge_data_csv(csv_dir: str) -> Tuple[List[str], List[str], list]:
 
 @enforce_types
 def challenge_data_csv_filename(csv_dir: str) -> str:
-    f = "challenge.csv"
+    f = "challenge_data.csv"
```

Comment:
I'm surprised this is being changed here. It's outside the scope of this issue.

And if this change isn't breaking things outside of df-py (in df-sql), maybe it's a sign that we need to test more throughly.